bpf/bpf_host.c

```diff
@@ -7,6 +7,8 @@
 #include <node_config.h>
 #include <ep_config.h>
 
+#define IS_BPF_HOST 1
```

Comment:
nit: maybe just use `#define IS_BPF_HOST` here since the check is `#if defined(IS_BPF_HOST)`

Reply:
That change seemed so obvious I didn't retest the patch after it. Bad idea.

Turns out the `1` is needed here because of how we define `is_defined`. I'll send a fix.

Reply:
Fix ready at https://github.com/cilium/cilium/pull/14255, with a test this time :slightly_smiling_face:

Reply:
Sorry about that!
It might make sense to replace `#if defined(IS_BPF_HOST)` with `#if IS_BPF_HOST == 1` at some point.